src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -144,7 +151,47 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(Items.MOSSY_COBBLESTONE_WALL).input(Items.COBBLESTONE_WALL).or().namedInput(Items.WHEAT_SEEDS).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.MOSSY_COBBLESTONE_WALL).input(Items.COBBLESTONE_WALL).or().namedInput(Items.MOSSY_COBBLESTONE).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.MOSSY_STONE_BRICKS).input(Items.STONE_BRICKS).or().input(Items.WHEAT_SEEDS).buildFor(consumer, Minestuck.MOD_ID);
-		CombinationRecipeBuilder.of(Items.CLAY).input(Items.SAND).and().input(Items.WATER_BUCKET).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.AMETHYST_CLUSTER).input(Items.GRASS_BLOCK).and().input(Items.AMETHYST_SHARD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.LARGE_AMETHYST_BUD).input(Items.SPYGLASS).and().input(Items.MEDIUM_AMETHYST_BUD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.MEDIUM_AMETHYST_BUD).input(Items.SPYGLASS).and().input(Items.AMETHYST_SHARD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.SMALL_AMETHYST_BUD).input(Items.SPYGLASS).or().input(Items.MEDIUM_AMETHYST_BUD).buildFor(consumer, Minestuck.MOD_ID);
```

Comment:
i think the logic behind it was something to the tune of 
Cluster = grass mound + amethyst shard
and then every stage of the bud is either shrinking up or down from medium with medium itself just being a bigger shard

Reply:
Spyglass definetely makes sense, although its best to only use it to represent only one of the two qualities. Either just magnifying or just shrinking. Just magnifying sounds like it makes the most sense given what a spyglass does

Reply:
that'll work

Reply:
These kinds of alchemy situations always end up a hassle, typically its cleaner just to ignore adding a combination since players will still be able to alchemize them as long as there is a cost